product_variant_cost_price/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Product Variant Cost
+====================
+
+Updates the product variant's cost when product template cost is updated.
```

Comment:
This is not as you explain. When i update the cost in template product, the variants are not updated. I can not update the product variant's cost  by hand because i have not the field Costing Method.